test/functional/rpc_deprecated.py

```diff
@@ -7,21 +7,19 @@
 
 class DeprecatedRpcTest(BitcoinTestFramework):
     def set_test_params(self):
-        self.num_nodes = 2
+        self.num_nodes = 1
         self.setup_clean_chain = True
-        self.extra_args = [[], ['-deprecatedrpc=bumpfee']]
+        self.extra_args = [[]]
 
     def run_test(self):
         # This test should be used to verify correct behaviour of deprecated
-        # RPC methods with and without the -deprecatedrpc flags. For example:
+        # RPC methods without the -deprecatedrpc flag. For example:
```

Comment:
This is the final comment after this change:

> This test should be used to verify correct behaviour of deprecated RPC methods without the -deprecatedrpc flag.

I think it can be reworded to be more explicit in the intention. I'm assuming the deprecated RPCs will just throw errors, please correct me if I'm wrong.

```
This test should be used to verify the errors of the currently deprecated RPC methods (without the -deprecatedrpc flag) until such RPCs are fully removed.
```

Suggesting this because first time I have looked at this test and I found it a little strange that it doesn't test any RPCs at the moment until I saw the file commit history.

Reply:
Added, thanks!